packages/server/graphql/handleGraphQLTrebuchetRequest.ts

```diff
@@ -38,24 +38,29 @@ const handleGraphQLTrebuchetRequest = async (
       return
     }
     try {
-      const result = await getGraphQLExecutor().publish({
-        docId,
-        query,
-        variables,
-        socketId,
-        authToken,
-        ip
+      return tracer.trace('handleGraphQLTrebuchetRequest', async (span) => {
```

Comment:
> Risks crashing our app because there is so much tracing

Did you measure any overhead that tracing adds?

Reply:
@mattkrick I see. Do you want to risk and deploy and see if it crashes? Or I can suggest to run some load tests (for example using https://k6.io/)